README.md

```diff
@@ -25,37 +25,40 @@
 <br />
 
 ## Install Giskard 🐢
-You can install the latest version of Giskard from PyPi using pip :
+Install the latest version of Giskard from PyPi:
 ```sh
-pip install giskard -U
+pip install "giskard[llm]"
 ```
 We officially support Python 3.9, 3.10 and 3.11.
 ## Try in Colab 📙
 [Open Colab notebook](https://colab.research.google.com/github/giskard-ai/giskard/blob/main/docs/getting_started/quickstart/quickstart_llm.ipynb)
 
 ______________________________________________________________________
 
-<p align="center">
-  <img src="https://raw.githubusercontent.com/giskard-ai/giskard/main/readme/architechture_giskard.png" alt="Giskard Architechture" width="800">
-</p>
-
-Giskard is a Python library that automatically **detects vulnerabilities** in AI models, from tabular models to LLM, including performance biases, data leakage, spurious correlation, hallucination, toxicity, security issues and many more.
-
-It's a powerful tool that helps data scientists **save time and effort** drilling down on model issues, and produce more **reliable and trustworthy models**.
+Giskard is a Python library that **automatically detects performance issues** in AI applications. The library covers LLM-based applications such as RAG agents, all the way to traditional ML models for tabular data. Issues detected include: 
```

Comment:
Instead of the sentence “The library covers… …data.”
I would write “The library encompasses a broad range of applications, from LLM (Large Language Model)-based systems like RAG (Retrieval-Augmented Generation) agents to conventional machine learning models designed for tabular data.”